packages/components/src/components/accordion/component.tsx

```diff
@@ -31,9 +31,43 @@ featureHint(`[KolAccordion] Tab-Sperre des Inhalts im geschlossenen Zustand.`);
 })
 export class KolAccordion implements API {
 	private readonly nonce = nonce();
+	private contentElement: HTMLElement | null = null;
+	private contentWrapperElement: HTMLElement | null = null;
+	private contentObserver = new ResizeObserver(this.resizeWrapper.bind(this));
```

Comment:
Nicht alle Features sind in den Testing-Libs entsprechende Mocks verfügbar. Du musst hierfür einen Workarround finden. Schau mal nach der nonce-Methode ;-)

Reply:
https://github.com/public-ui/kolibri/blob/a99f3d949eac6884dacc1da09a44e6aaaccfb8e6/packages/components/src/utils/dev.utils.ts#L183